### ISSUE TITLE
 Audit #1 – Login Page – Error message on wrong input not programmatically connected to inputs

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4323]
Connected to Audit 1, see #4300 
1.1 Login Page, URL: https://ocis-a11y.owncloud.works/

# Issue

On entering wrong account data or missing a required field a text error message does exist, but is not programmatically connected to the invalid field.

# Remediation

Technique ARIA21: Using both `aria-invalid` and `aria-describedby` on the invalid field(s), where the latter points via IDREF to the error message, which in itself has said ID https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
